README.md

```diff
@@ -22,4 +22,7 @@ docker-compose up
 ### Some Specifications for the App
 this app is built with the following libraries/packages:
 - [Gin Web framework](https://github.com/gin-gonic/gin/) for api routing
-- [AWS SDK](https://aws.amazon.com/sdk-for-go/) for AWS S3
```

Comment:
this commit contains two different changes:
1. update the readme
2. delete s3_test

in this case it's better to slip the commit in two: it will make the rebase easier and you'll be able to remove the commit "s3 mocck test" and "#delete s3_test".
Keep the story in your `git log` clean it's useful to quickly understand the history of your work! :)